src/hooks/api/exchange-rates/p2p-exchange-rates/useP2PExchangeRates.ts

```diff
@@ -0,0 +1,29 @@
+import { useEffect, useState } from 'react';
+import { useExchangeRates } from '@deriv-com/api-hooks';
+
+const BASE_CURRENCY = 'USD';
+
+const useP2PExchangeRates = (targetCurrency: string) => {
```

Comment:
Can we add a small description of this hook? 🙏

Reply:
ah yea sure 🐸